appveyor.yml

```diff
@@ -96,8 +96,9 @@ build_script:
 - cd %ROOT_DIRECTORY%
 - mkdir build && cd build
 - if defined cmake (
-  cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="/WX" -G "Visual Studio 14 2015 Win64" .. &&
-  cmake --build .
+  set "PATH=c:\Python27-x64;%PATH%" &&
+  cmake -DCMAKE_CXX_FLAGS="/WX /wd4244 /wd4459" -Dpython-bindings=%python% -Dboost-python-module-name="python" -Dskip-python-runtime-test=true -DPython_ADDITIONAL_VERSIONS="2.7" -G "Visual Studio 14 2015 Win64" .. &&
+  cmake --build . --config Release
```

Comment:
this `--config` flag, is it an msvc-only option? It looks a lot better than `-DCMAKE_BUILD_TYPE=`. Could it be used on travis as well?

Reply:
No, they differ. There are two types of generators in CMake: single configuration and multi-configuration. The first ones (e.g. Makefile) are controlled by the regular CMake variable during configuration phase (`CMAKE_BUILD_TYPE`, actually just selects a predefined set of compiler and linker flags), those of the second type (e.g. MSVS) have their own configurations inside and the `--config` switch is used when invoking build (i.e. when configuration stage has finished).